src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -123,4 +123,8 @@ public static final class IntakeConstants {
     // TODO: update placeholder roller Motor ID
     public static final int kMotorID = -1;
   }
+
+  public class Bindings {
```

Comment:
What kind of bindings?

Reply:
Responded to

Reply:
The class should be self describing. Call it `ButtonBindings` and make a javadoc for it.

Reply:
Also move to `RobotConfig` and call it `IntakeBindings`. Also constants should be prefixed with `k`